docs/tutorials/02_migrating_to_renku.rst

```diff
@@ -94,19 +94,24 @@ Now your project is ready to be sent to the web platform!
 Push your project to renkulab
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Pushing your project to Renku's GitLab instance boils down to adding the right
-remote and pushing the code. Perform the following, replacing ``<namespace>`` with
-the username that appears on the renkulab-generated GitLab instance (unless you
-are pushing to a group), and ``<project-name>`` with the name you are giving your project.
+Before you can push your project to Renku you need to create a GitLab repository
```

Comment:
AFAIK you can create a project on GitLab by simply pushing, no need to create the the project through the UI first. However, the information about access management etc through gitlab is still very useful.

Reply:
Actually reading the PR description now - sorry about that.